scripts/walk-the-dog.sh

```diff
@@ -157,7 +160,29 @@ then
     echo 'Deploying AKS'
     echo '****************************************************'
 
-    deploy_azure_kubernetes_service
+    deploy_azure_kubernetes_service # from functions.sh
+
+    # connect to AKS cluster and deploy namespace, configmap
+    echo ''
+    echo "connect to AKS cluster and deploy namespace, configmap"
+    az aks get-credentials --resource-group $RG --name $AKS_NAME
+    kubectl create ns reddog
+    CONFIGMAP_FILE=../outputs/config-map-reddog-java-spring-$SUFFIX.yaml
+    kubectl apply -f $CONFIGMAP_FILE
+
+    # deploy Flux configuration with AKS extension
+    echo ''
+    echo "deploy Flux configuration with AKS extension"
+    az k8s-configuration flux create \
+        --resource-group $RG \
+        --cluster-name $AKS_NAME \
+        --cluster-type managedClusters \
+        --scope cluster \
+        --name reddog-java-apps \
+        --namespace flux-system \
+        --url https://github.com/appdevgbb/reddog-code-spring.git \
```

Comment:
We will likely need to update this once we move this under `Azure-samples`, but we will get there when we get there.